components/organisms/InsightPage/InsightPage.tsx

```diff
@@ -383,6 +395,18 @@ const InsightPage = ({ edit, insight, pageRepos }: InsightPageProps) => {
     return <></>;
   };
 
+  const getOrganizationLookupError = (code: OrgLookupError) => {
+    if (code === OrgLookupError.Error) {
+      return <Error errorMessage="There was error retrieving this organization's public repositories." />;
+    }
+
+    if (code === OrgLookupError.Invalid) {
+      return <Error errorMessage="This organization entered is invalid." />;
+    }
+
+    return <></>;
```

Comment:
refactor (blocking): return null

```suggestion
    return null;
```